src/components/atoms/SfCheckbox/README.md

```diff
@@ -0,0 +1,83 @@
+# SfCheckbox
+
+<!-- Write about general purpose of the component. Include screenshot (to be replaced with a live example once we migrate to vuepress) -->
+
+## HTML Template
+
+#### Basic
+````html
+<sf-checkbox
+    id="sf-checkbox-default"
+    name="sf-checkbox-default"
+    v-model="status"
+/>
+````
+
+#### Advanced (slots)
+````html
+<sf-checkbox
+    id="sf-checkbox-slots"
+    name="sf-checkbox-slots"
+    v-model="status"
+>
+    <template #label>
+      <label for="sf-checkbox-slots" class="sf-checkbox__label">
+        <span class="sf-checkbox__icon">
+          <img
+            class="sf-checkbox__icon-svg"
+            src="/assets/check.svg"
+            alt="Checkbox Icon"
+          />
+        </span>
+        <img src="https://via.placeholder.com/350x60?text=Checkbox+Content+Image" alt="placeholder image" style="display: block;">
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Animi dicta ea fugiat maiores non, nulla praesentium similique voluptatum! Aperiam asperiores dolores eius eum impedit minima necessitatibus numquam saepe ullam veritatis!
+      </label>
+    </template>
+</sf-checkbox>
+````
+
+## Props
+
+- `finalValue` - Final value 
+- `uncheckedValue` - Unchecked value - if the checkbox is unchecked, this is the emitted input value (uncheckedValue), instead of 'value'
+- `value` - Input value 
+- `id` - Input id 
+- `name` - Input name 
+- `iconClass` - Class for icon 
+- `labelClass` - Class for label 
+- `inputClass` - Class for input 
```

Comment:
not sure about classes as props, imho we should handle this as possible with slots.
actually, label and icon may be replaced with `label` slot, maybe you can add more one slot just for the icon, so `labelClass` and `inputClass` can be removed

Reply:
Exactly. We use scss modifiers for whole component handling most common use cases (check banner and button) and SCSS vars for less common modifications

Reply:
Hi guys,

as mentioned above, i've copied this component from the Alpaca storybook, hence majority of the the API, HTML + SCSS being whatever they wrote, with minor amends, (see initial first post above where i put " I've copied the JS, SCSS and HTML from Alpaca as advised"), i did that because i thought that was what was meant to be done instead of rewriting it? 
![image](https://user-images.githubusercontent.com/2095899/55937797-c6349b00-5c31-11e9-966e-36ec7a59a598.png)
Where in the tutorial for SFUI it says "markup should always be copied (never imported) and alpaca classes should be repalced with same ones but prefixed with sf-"

I can always rewrite these from scratch in a different way with your changes in mind, however i've just simply ported this from Alpaca over to SFUI - As i was under the impression that's what we was meant to be doing (from the guides), if so the guides may need to be clarified.

Reply:
Yeah I needed to be more precise. Some of Alpaca components are not yet adjusted to the format we agreed on. You;'re right and I need to clarify those

Reply:
Ok, maybe we can discuss the changes that you are going to do and i can perhaps clarify the docs from that?

Reply:
I had read that you copied the component, I should also have mentioned it in my review, sorry for this...
You can fix the details in the SCSS as suggested? And what do you think about the suggestions we made for the component API?

Reply:
Yeah i can amend the SCSS and also make the changes to the API and see what you think, i'll rewrite it and see just add what we need as much of is going to change anyway, like converting it over to slots instead of props mainly